### PR TITLE
Add an alert edit UI, refactor paginator

### PR DESCRIPTION
**Pull request by BrandonArp**

### CODE REVIEW

File: app/assets/javascripts/classes/alerts/EditAlertViewModel.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2016 Smartsheet.com
```
Comment (vjkoskela) [inline app/assets/javascripts/classes/alerts/EditAlertViewModel.ts]: 2017

File: conf/db/migration/metrics_portal_ddl/h2/V10__fix_pg_etag_triggers.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2016 Smartsheet.com
```
Comment (vjkoskela) [inline conf/db/migration/metrics_portal_ddl/h2/V10__fix_pg_etag_triggers.sql]: 2017

File: conf/db/migration/metrics_portal_ddl/postgresql/V10__fix_pg_etag_triggers.sql [hunk 3]
```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2016 Smartsheet.com
```
Comment (vjkoskela) [inline conf/db/migration/metrics_portal_ddl/postgresql/V10__fix_pg_etag_triggers.sql]: 2017

File: public/html/alerts/EditAlertViewModel.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<!--
+  ~ Copyright 2016 Smartsheet.com
```
Comment (vjkoskela) [inline public/html/alerts/EditAlertViewModel.html]: 2017

File: public/html/components/Pager.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<!--
+  ~ Copyright 2016 Smartsheet.com
```
Comment (vjkoskela) [inline public/html/components/Pager.html]: 2017

File: app/assets/javascripts/classes/alerts/EditAlertViewModel.ts (hunk 1 above)
Comment (BrandonArp) [reply app/assets/javascripts/classes/alerts/EditAlertViewModel.ts]: fixed

File: conf/db/migration/metrics_portal_ddl/h2/V10__fix_pg_etag_triggers.sql (hunk 2 above)
Comment (BrandonArp) [reply conf/db/migration/metrics_portal_ddl/h2/V10__fix_pg_etag_triggers.sql]: fixed

File: conf/db/migration/metrics_portal_ddl/postgresql/V10__fix_pg_etag_triggers.sql (hunk 3 above)
Comment (BrandonArp) [reply conf/db/migration/metrics_portal_ddl/postgresql/V10__fix_pg_etag_triggers.sql]: fixed

File: public/html/alerts/EditAlertViewModel.html (hunk 4 above)
Comment (BrandonArp) [reply public/html/alerts/EditAlertViewModel.html]: fixed

File: public/html/components/Pager.html (hunk 5 above)
Comment (BrandonArp) [reply public/html/components/Pager.html]: fixed

**Pull request merged**